parsers/MN.py

```diff
@@ -143,6 +145,46 @@ def fetch_consumption(
     return consumption_list.to_list()
 
 
+def fetch_exchange(
+    zone_key1: ZoneKey,
+    zone_key2: ZoneKey,
+    session: Session = Session(),
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+) -> list[dict]:
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates.")
+
+    sorted_zone_keys = ZoneKey("->".join(sorted([zone_key1, zone_key2])))
+
+    if sorted_zone_keys != "CN->MN":
+        raise NotImplementedError(
+            "This parser is only able to fetch CN->MN exchange data."
+        )
+
+    query_data = query(session, logger, ZoneKey("MN"))
+
+    russia_data = ru_fetch_exchange("MN", "RU-2", session, logger=logger)
+
+    exchange_list = ExchangeList(logger)
+
+    for data in russia_data:
+        if (
+            data["datetime"] == query_data["time"]
+            and data["sortedZoneKeys"] == "MN->RU-2"
+            and "netFlow" in data.keys()
+        ):
+            exchange_list.append(
+                datetime=query_data["time"],
+                zoneKey=sorted_zone_keys,
+                # We calculate the flow with the total imports and the MN->RU-2 exchange, as Mongolia has only two connections
+                netFlow=query_data["importMW"] + data["netFlow"],
+                source="https://ndc.energy.mn/",
```

Comment:
Not sure if you know this but we have created generic functions to do this and ensure everything match. You can asses it with
```python
exchange_list.merge_exchanges()
```
The only issue is that it only accepts a list of `ExchangeList` object to merge.

Reply:
I am not sure if I have understood correctly, but isn't this function meant to merge ExchangeLists of the same sortedzones? In this case I am substracting the russian connection to the total.